crates/ruma-client-api/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Breaking changes:
   clients that required it.
 - The `reason` field of `report_room::v3::Request` is now required, due to a
   clarification in the spec.
+- The `authentication` field of `discovery::discover_homeserver::Response` has
```

Comment:
It's not considered a breaking change, it was behind an unstable flag. You can put it in the "Improvements" section.

Reply:
Ah ok, TIL! It is now under Improvements. 👌